cmd/argoexec/commands/emissary.go

```diff
@@ -132,6 +132,18 @@ func NewEmissaryCommand() *cobra.Command {
 				command.Stderr = io.MultiWriter(os.Stderr, stderr)
 			}
 
+			if _, ok := os.LookupEnv("DEBUG_PAUSE_BEFORE"); ok {
+				for {
+					// User can create the file: /ctr/NAME_OF_THE_CONTAINER/continue/before
+					// in order to break out of the sleep and release the container from
+					// the debugging state.
+					if _, err := os.Stat(varRunArgo + "/ctr/" + containerName + "/continue/before"); os.IsNotExist(err) {
```

Comment:
To use this I have to create directory and file? Two actions.  Could it be instead a single `touch`?

Reply:
Sure we can drop the `continue` folder.